ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/materialization/ks/KsLocator.java

```diff
@@ -140,6 +141,11 @@
     return partitionLocations.build();
   }
 
+  @SuppressFBWarnings(value = "EI_EXPOSE_REP")
```

Comment:
Added this to throw more meaningful exceptions which are also helpful for debugging when there are multiple hosts